debezium-connector-db2/src/main/java/io/debezium/connector/db2/Db2DatabaseSchema.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.db2;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.relational.HistorizedRelationalDatabaseSchema;
+import io.debezium.relational.Table;
+import io.debezium.relational.TableId;
+import io.debezium.relational.TableSchemaBuilder;
+import io.debezium.relational.ddl.DdlParser;
+import io.debezium.relational.history.TableChanges;
+import io.debezium.schema.SchemaChangeEvent;
+import io.debezium.schema.SchemaChangeEvent.SchemaChangeEventType;
+import io.debezium.schema.TopicSelector;
+import io.debezium.util.SchemaNameAdjuster;
+
+/**
+ * Logical representation of SQL Server schema.
+ *
+ * @author Jiri Pechanec
+ */
+public class Db2DatabaseSchema extends HistorizedRelationalDatabaseSchema {
```

Comment:
Let's discuss the need for schema history; it's not quite clear to me yet whether/how it applies to DB2.